src/run-scenario.jl

```diff
@@ -56,7 +56,7 @@ function run_scenario(
 
     @timeit to "save_solution!" save_solution!(energy_problem)
 
-    if output_folder != ""
+    if output_folder != "" && energy_problem.solved
```

Comment:
Since you mentioned, how about adding an `@warn` here? Like

```julia
if output_folder != ""
    if energy_problem.solved
        ...
    else
        @warn "..."
    end
end
```